src/java.base/share/classes/java/nio/file/CopyMoveHelper.java

```diff
@@ -109,21 +109,21 @@ static void copyToForeignTarget(Path source, Path target,
         LinkOption[] linkOptions = (opts.followLinks) ? new LinkOption[0] :
             new LinkOption[] { LinkOption.NOFOLLOW_LINKS };
 
-        // retrieve source posix view, null if unsupported
-        final PosixFileAttributeView sourcePosixView =
-            Files.getFileAttributeView(source, PosixFileAttributeView.class);
```

Comment:
I don't think we should delete this. Instead, I think we can initialize sourceSupportsPosixAttributes with `Files.getFileAttributeView(source, PosixFileAttributeView.class) != null`.

Reply:
> I don't think we should delete this

So changed in ad29084.